src/xebia_email_signature/templates/signature.html.jinja

```diff
@@ -7,7 +7,7 @@
         <tr style="line-height: 1;">
         {% if data.get('profile_picture') %}
         <td>
-        <img src="data:image/png;base64,{{ data['profile_picture'] }}"/>
+        <img src="data:image/png;base64,{{ data['profile_picture'] }}" width="60px" height="60px" />
```

Comment:
@rdeveen  de width and height should not be specified. The image should be in the desired format.

Reply:
So, the profile image should be generated in 60x60 if that is what you need. In the xebia design it is 90x90